### DISCUSSION TITLE
How can I create my own custom datatype in pydantic?

### DISCUSSION THREAD

**Discussion by Mitra-Electronics** [pydantic/pydantic#3366] [Question] [answered]
I want to create my own datatype to validate countries without using pydantic validator decorator since there are a long list of countries and keep on changing.

**Comment by uriyyo** [comment, marked as answer]
You can find information [here](https://pydantic-docs.helpmanual.io/usage/types/#custom-data-types)

**Comment by Mitra-Electronics** [comment]
Ok thanks
